c++ training/SteamProject/Videogame/Steam.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "CategoryContainer.h"
+
+
+
+class FSteam {
+private:
+	FCategoryContainer CategoryContainer;
+	FCategory Uncategorized = FCategory("Uncategorized");
+
+	int NumberOfGames;
+
+	enum class MenuCommand
```

Comment:
according to the coding standards, the enums have to be named with the prefix `E`